app/Main.hs

```diff
@@ -18,4 +18,4 @@ readEvalPrint chordStr =
     let xs = Parser.parseChord chordStr
     _ <- print xs
     _ <- print $ CanonicalChord.canonicalizeChord <$> xs
-    print $ show <$> Lib.chordToNotes <$> CanonicalChord.canonicalizeChord <$> xs
+    print $ show . Lib.chordToNotes . CanonicalChord.canonicalizeChord <$> xs
```

Comment:
As it turns out, `<$>` for functions is function composition; it's clearer to write compositions as compositions rather than supposing that the reader is familiar enough with this functor to recognize that this is the case.